var/spack/repos/builtin/packages/r-dupradar/package.py

```diff
@@ -6,17 +6,18 @@
 from spack.package import *
 
 
-class BioconductorDupradar(RPackage):
+class RDupradar(RPackage):
     """Assessment of duplication rates in RNA-Seq datasets"""
 
     homepage = "https://bioconductor.org/packages/3.16/bioc/html/dupRadar.html"
     url = "https://bioconductor.org/packages/release/bioc/src/contrib/dupRadar_1.30.0.tar.gz"
     maintainers("pabloaledo")
 
-    bioc = "dupradar"
+    bioc = "dupRadar"
 
     version("1.30.0", sha256="a299d7a4578047dfc19237e34255b0f50f70ce41d29762ef9f5a7741ba35aa3d")
+    version("1.32.0", commit="7e07fc3a3901f8cae0203759fc24dd7df430a07f")
 
     depends_on("r-kernsmooth")
```

Comment:
as per https://spack.readthedocs.io/en/latest/build_systems/rpackage.html#build-system-dependencies, 

please for all 4 recipes explicitly state R dependency as
```
depends_on('r', type=('build', 'run'))
```

eventually specifying version constraints for those packages that have them (eg `'r@M.m:'`) - see package homepage `https://bioconductor.org/packages/PACKAGENAME`,  **Depends** table entries